### PR TITLE
chore: add svd to paddle linalg

### LINKED ISSUE #17843: svd
Implement the paddlepaddle frontend for the function svd (Linear Algebra)

### PR DESCRIPTION
**Pull request by a0m0rajab**
closes #17843

### CODE REVIEW

**Comment by abdulasiraj** [comment]
Hi @a0m0rajab,
Please resolve the conflicts!
Thanks!

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -2968,3 +2968,39 @@ def test_paddle_log1p(
         frontend=frontend,
         on_device=on_device,
     )
+
+# svd
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
```
Comment (abdulasiraj) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2974]: here you should write the function name not the CLASS_TREE

**Comment by a0m0rajab** [comment]
@abdulasiraj conflict resolved.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py
Comment (abdulasiraj) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:1]: Hi @a0m0rajab,
Thanks for great work. just one quick thing, please add the test for this in `test_linalg.py` file. Thanks!

**Review by abdulasiraj — CHANGES_REQUESTED** [review]
requested review!

**Comment by a0m0rajab** [comment]
Thanks, the file moved.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_linalg.py [hunk 2]
```diff
@@ -839,3 +839,40 @@ def test_paddle_dist(
         y=x[1],
         p=p,
     )
+
+
+# svd
+@handle_frontend_method(
```
Comment (abdulasiraj) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_linalg.py:845]: It should be like `handle_frontend_test` and also you've to change the parameters accordingly. Also please make sure, All tests are being passed. Thanks!

**Review by abdulasiraj — CHANGES_REQUESTED** [review]

**Comment by Aarsh2001** [comment]
Hey @a0m0rajab, closing this PR due to inactivity, feel free to reopen once you're done with your changes. Thanks

**Pull request closed without merging**